lib/blacklight.rb

```diff
@@ -53,9 +54,10 @@ def self.create_canvas_course(resources, zip_name)
     course
   end
 
-  def self.initialize_course(filename)
-    metadata = Blacklight::CanvasCourse.metadata_from_file(filename)
-    course = Blacklight::CanvasCourse.from_metadata(metadata)
-    course.upload_content(filename)
+  def self.initialize_course(canvas_import, blackboard_export)
+    metadata = Blacklight::CanvasCourse.metadata_from_file(canvas_import)
+    bb_zip = Zip::File.new(blackboard_export) unless blackboard_export.nil?
```

Comment:
Do you mean Zip::File.open()? It doesn't seem that read will do what I need it to do.